test/e2e/infra/secondary_network/secondary_network_prerequisite.sh

```diff
@@ -0,0 +1,361 @@
+        --sriov-vf-count <count>                Number of virtual functions to be created on the base network interface.
+        --host-name <k8s control node name/IP>  K8s cluster control node's host name or IP address."
+
+_ssh_config_info="Prerequisite: Kubernetes(K8s) cluster is up and running at the local or remote server.
+
+Follow these steps to configure SSH information to access the kubernetes cluster control node.
+
+1) Update remote/local cluster control node information at your /home/<user>/.ssh/config
+Example:
+Host <Control-Plane-Node>
+    HostName <Control-Plane-IP>
+    Port 22
+    user ubuntu
+    IdentityFile /home/ubuntu/.ssh/id_rsa
+
+2) Add remote/local cluster control node's public key (id_rsa.pub) to /home/<user>/.ssh/authorized_keys
+
+3) Set PubkeyAuthentication at sshd_config to yes at your remote/local cluster control node
+
+4) Esure SSH to the remote/local node works after the above configuration (ssh <control node/host name>)"
+
+function echoerr {
+        echo >&2 "$@"
+}
+
+function print_usage {
+	echoerr "$_usage"
+}
+
+function print_ssh_config_info {
+	echoerr "$_ssh_config_info"
+}
+
+function print_help {
+	echoerr "Try '$0 --help' for more information."
+}
+
+SRIOV_PLUGIN_REL_TAG=v3.5.1
+WHEREABOUTS_CNI_REL_TAG=v0.5.4
+export URL_ANTREA_CONFIG_MAP="https://raw.githubusercontent.com/k8snetworkplumbingwg/sriov-network-device-plugin/$SRIOV_PLUGIN_REL_TAG/deployments/configMap.yaml"
+export URL_SRIOV_DP_DAEMONSET="https://raw.githubusercontent.com/k8snetworkplumbingwg/sriov-network-device-plugin/$SRIOV_PLUGIN_REL_TAG/deployments/k8s-v1.16/sriovdp-daemonset.yaml"
+export URL_WHEREABOUTS_IP_POOLS="https://raw.githubusercontent.com/k8snetworkplumbingwg/whereabouts/$WHEREABOUTS_CNI_REL_TAG/doc/crds/whereabouts.cni.cncf.io_ippools.yaml"
+export URL_WHEREABOUTS_OVERLAPPING_IP_RANGE_RES="https://raw.githubusercontent.com/k8snetworkplumbingwg/whereabouts/$WHEREABOUTS_CNI_REL_TAG/doc/crds/whereabouts.cni.cncf.io_overlappingrangeipreservations.yaml"
+export YAML_CONFIG_MAP='configMap.yaml'
+export YAML_SRIOV_DP_DAEMONSET='sriovdp-daemonset.yaml'
+export YAML_ANTREA='antrea.yml'
+export YAML_WHEREABOUTS_IP_POOL='whereabouts.cni.cncf.io_ippools.yaml'
+export YAML_WHEREABOUTS_OVERLAPPING_IP_RANGE_RES='whereabouts.cni.cncf.io_overlappingrangeipreservations.yaml'
+export YAML_NET_ATTACH_DEF_CRD='network-attachment-definition-crd.yml'
+export YAML_VIRTUAL_NET_INSTANCE='virtual-network-instance-crd.yml'
+#SEC_NET_LOCAL_CONFIG_DIR_PATH is at the test execution server where the required config files are downloaded/generated.
+export SEC_NET_LOCAL_CONFIG_DIR_PATH=$THIS_DIR/secondary_network_config/
+#SEC_NET_REMOTE_CONFIG_DIR_PATH is at the K8s cluster's control node where we configure the cluster for secondary network.
+export SEC_NET_REMOTE_CONFIG_DIR_PATH='/tmp/secondary_network_config/'
+export NAMESPACE='kube-system'
+
+interface=''
+vfs_num=''
+action=''
+configure=''
+host_name_set=false
```

Comment:
I don't think we need `host_name_set`, when we can just check that `hostIP != ""` to the same effect

Reply:
I have modified it accordingly to check for hostIP in validate() now. But, I do not see a need for it though, as we can see below, if the host-name was not provided, it fails at the arg processing loop itself.

./secondary_network_prerequisite.sh --setup --host-name
/home/kcontrol/Antrea_9th_August2022/antrea/test/e2e/infra/secondary_network /home/kcontrol/Antrea_9th_August2022/antrea/test/e2e/infra/secondary_network
**--host-name <value> not provided.**
Usage: ./secondary_network_prerequisite.sh [--setup] [--cleanup]
                  [--base-interface <interface name>] [--sriov-vf-count <count>] --host-name <name>

Deploy Antrea CNI and its native secondary network prerequisites on an existing/running K8s cluster control node.

        --setup                                 Deploy all the pre-requisite plugins and CRDs for secondary network configuration.
        --cleanup                               Remove all the pre-requisite plugins and CRDs deployed for secondary network configuration.
        --base-interface <interface name>       Network interface to create VFs.
        --sriov-vf-count <count>                Number of virtual functions to be created on the base network interface.
        --host-name <k8s control node name/IP>  K8s cluster control node's host name or IP address.